### PR TITLE
Adding virtualenv as a package

### PR DESCRIPTION
**Pull request by stevenuray**

### CODE REVIEW

File: packages/virtualenv.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+description: "Virtual "
```
Comment (BlackVegetable) [inline packages/virtualenv.yaml]: Just Virtual?

Comment (stevenuray) [reply packages/virtualenv.yaml]: lol, whoops, nice catch

File: packages/virtualenv.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+description: "virtualenv"
+webPage: https://virtualenv.pypa.io/en/stable//
```
Comment (earlye) [inline packages/virtualenv.yaml]: I'd drop everything after `.io/`. That will redirect to `/en/stable/`, is less likely to change over time, and in theory allows folks in other locales to use kyrios.

**Review by earlye — CHANGES_REQUESTED** [review]
Just a nit, really.

File: packages/virtualenv.yaml (hunk 2 above)
Comment (stevenuray) [reply packages/virtualenv.yaml]: Makes sense to me, I'll change that.

**Review by earlye — APPROVED** [review]
LGTM

**Pull request merged**